pages/components/docs/getting-started.md

```diff
@@ -1 +1,49 @@
-### Coming soon
+# Getting Started
+
+To get started using Primer Components, run `npm i @primer/components` in your application.
+
+You can now start importing Primer Components! There are a few ways to import Primer Components. You can either:
+
+Import them individually from the main bundle
+
+```
+import {Box, Flex} from '@primer/components'
+```
+
+or, if you've configured your application to tree-shake with webpack, you can import them indivdually from the `src` folder:
+
+```
+import Box from '@primer/components/src/Box'
+import Flex from '@primer/components/src/Flex'
```

Comment:
Should we be recommending that people import components like this? A few people in the #react channel have had trouble with this kind of import because the imported code isn't processed by Babel.

In theory, we should be able to make tree-shaking work with imports like `import {Box, Flex} from '@primer/components'`, right?

Reply:
That was my assumption but webpack is still pulling in all exports even though our ESM module should work with tree-shaking fine. You can follow the discussion here: https://github.com/primer/components/issues/456